lib/validator/validator.go

```diff
@@ -326,8 +329,26 @@ func validateIPPool(v *validator.Validate, structLevel *validator.StructLevel) {
 				}
 			}
 		}
+
+		// IP Pool CIDR cannot overlap with IPv4 or IPv6 link local address range.
+		_, ipv4LinkLocalNet, _ := net.ParseCIDR("169.254.0.0/16")
```

Comment:
can't have this type as a const see: https://golang.org/ref/spec#Constants so putting it in the `var` section